server/middleware/bookVisitSessionValidator.ts

```diff
@@ -0,0 +1,67 @@
+import { RequestHandler, Response } from 'express'
+import createError from 'http-errors'
+import paths from '../constants/paths'
+import logger from '../../logger'
+
+const journeyOrder: string[] = [
```

Comment:
Idea here is to set journey order in an array so the numerical 'stage' in the journey can be calculated by comparing the array indices of the current & test paths.

So, quite similar to what `stage` is doing in the equivalent Staff UI middleware ([server/middleware/sessionCheckMiddleware.ts](https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/blob/main/server/middleware/sessionCheckMiddleware.ts)) but here the middleware calculates the stage itself rather than having it passed in per route. I'm hoping this might be simpler to maintain if routes are added/changed.